wallet/db.c

```diff
@@ -838,7 +838,7 @@ void *sqlite3_column_arr_(const tal_t *ctx, sqlite3_stmt *stmt, int col,
 bool sqlite3_bind_short_channel_id(sqlite3_stmt *stmt, int col,
 				   const struct short_channel_id *id)
 {
-	char *ser = short_channel_id_to_str(id, id);
+	char *ser = short_channel_id_to_str(NULL, id);
```

Comment:
Should not *this* particular change be done with the previous commit instead?  Otherwise previous commit, may fail when that commit is used on an old db in production env (though of course it should not be used in production, still, the principle of it is...).

Reply:
Agreed, I'll move it!